tests/snapshots/tests__pre_reduce_run@sort__radix__radix_sort_ctr.hvmc.snap

```diff
@@ -0,0 +1,20 @@
+---
+source: tests/tests.rs
+expression: output
+input_file: examples/sort/radix/radix_sort_ctr.hvmc
+---
+#549755289600
+pre-reduce:
+RWTS   :     988_283_046
+- ANNI :     549_453_877
+- COMM :      53_477_380
+- ERAS :     203_423_749
+- DREF :     123_207_775
+- OPER :      58_720_265
+run:
+RWTS   :               0
```

Comment:
looks like the pre-reduce test isn't properly excluding `main` here; @FranchuFranchu can you fix this? (in a separate PR)